docs/getting_started/install_manually.md

```diff
@@ -1,29 +1,284 @@
+!!! caution "Demo installation"
+
+    The Demo is intended for learning and inspiration. Do not use it as a basis for actual projects.
+
+### eZ Platform installation types
+
+| Type | Repository |
+|------|----------------|
+| `clean` | [ezplatform](https://github.com/ezsystems/ezplatform) |
+| `platform-demo` | [ezplatform-demo](https://github.com/ezsystems/ezplatform-demo) |
+
+### eZ Platform Enterprise Edition installation types
+
+| Type | Repository |
+|------|----------------|
+| `studio-clean` | [ezplatform-ee](https://github.com/ezsystems/ezplatform-ee) |
+| `platform-ee-demo`  | [ezplatform-ee-demo](https://github.com/ezsystems/ezplatform-ee-demo) |
+
+## Installing on Linux
+
+## 1. Install a LAMP Stack
+
+Depending on your Linux distribution, you may need to install part or all of the LAMP (Linux, Apache, MySQL, PHP) stack required to run eZ Platform.
+Before getting started, make sure you review the [requirements](requirements_and_system_configuration.md) page to see the systems we support and use for testing.
+
+You may use your system's package manager (yum, apt-get, etc.) to obtain a copy of Apache, MySQL, and PHP, or download the latest versions from the official websites and install manually:
+
+-   [Apache](https://httpd.apache.org/download.cgi)
+-   [MySQL](http://dev.mysql.com/downloads/mysql/)
+-   [PHP 5.6+](http://php.net)
+
+For example, with Debian you can use `apt-get` to install `apache2`, `mysql-server`, `mysql-client`,
+and all PHP packages listed in the [requirements](requirements_and_system_configuration.md).
+You also need `git` for version control.
+
+!!! note "Set up Swap on Debian"
+
+    If your machine only has 1 or 2 GB of RAM, be sure to set up swap so you don't run out of RAM when running the composer scripts later on.
+
+    Swap space allows your system to utilize the hard drive to supplement capacity when RAM runs short. Composer install will fail if there is insufficient RAM available, but adding swap will allow it to complete installation.
+
+    Via the command line, you can set up and enable swap on your Debian machine via the following commands (as root):
+
+    ``` bash
+    fallocate -l 4G /swapfile
+    chmod 600 /swapfile
+    mkswap /swapfile
+    swapon /swapfile
+    echo "/swapfile   none    swap    sw    0   0" >> /etc/fstab
+    sysctl vm.swappiness=10
+    echo "vm.swappiness=10" >> /etc/sysctl.conf
+    sysctl vm.vfs_cache_pressure=50
+    echo "vm.vfs_cache_pressure=50" >> /etc/sysctl.conf
+    ```
+
+## 2. Get Composer
+
+a. Install Composer, the PHP command line dependency manager, by running the following command in the terminal:
+
+``` bash
+php -r "readfile('https://getcomposer.org/installer');" | php
+```
+
+b. Move the downloaded `composer.phar` file to a globally-available path:
+
+``` bash
+mv composer.phar /usr/local/bin/composer
+```
+
+## 3. Download eZ Platform
+
+You can download eZ Platform in two ways: by downloading an archive, or cloning the GitHub repository:
```

Comment:
`by downloading an archive, or cloning the GitHub repository:` - redundant sentence as it is explained in points a and b below. I would remove it